test/mocks/MockVatSubscriptionService.scala

```diff
@@ -44,6 +44,13 @@ trait MockVatSubscriptionService extends MockitoSugar with BeforeAndAfterEach {
     )(ArgumentMatchers.any[HeaderCarrier],
       ArgumentMatchers.any[ExecutionContext])) thenReturn response
 
+  def mockRemoveWebsiteAddress(vrn: String, website: String)(response: Future[UpdatePPOBResponse]): Unit =
```

Comment:
This is almost exactly the same as the function below it. You could just re-use that one in your tests.